templates/vault/0/docker-compose.yml.tpl

```diff
@@ -0,0 +1,36 @@
+version: '2'
+volumes:
+  vault-config:
+    driver: ${volumeDriver}
+  vault-file:
+    driver: ${volumeDriver}
+services:
+  vault-lb:
+    image: rancher/lb-service-haproxy:v0.7.15
+    ports:
+    - 8200:8200/tcp
+    - 8201:8201/tcp
+    labels:
+      io.rancher.scheduler.affinity:host_label: lbhost=true
+      io.rancher.container.agent.role: environmentAdmin,agent
+      io.rancher.container.agent_service.drain_provider: 'true'
+      io.rancher.container.create_agent: 'true'
+  vault:
+    cap_add:
+    - IPC_LOCK
+    image: vault

```

Comment:
Please, set a concrete tag for docker images. Generic tags, implicit or explicit are not reproducible.
```
-    image: vault
+    image: vault:0.9.6
```

Reply:
Done!